src/js/setup.js

```diff
@@ -31,6 +31,7 @@ const autoSetup = function() {
   // through each list of elements to build up a new, combined list of elements.
   const vids = document.getElementsByTagName('video');
   const audios = document.getElementsByTagName('audio');
+  const divs = document.getElementsByTagName('video-js');
```

Comment:
we now look for `video-js` elements for `data-setup`, and so we *definitely* can keep `data-setup` now.